sympy/simplify/simplify.py

```diff
@@ -1258,3 +1258,42 @@ def _real_to_rational(expr, tolerance=None):
                     r = Integer(0)
         reps[key] = r
     return p.subs(reps, simultaneous=True)
+
+
+def clear_coefficients(expr):
+    """Return `p, (R, f(R))` where `p` is the expression obtained
```

Comment:
I'm thinking this would be simpler to let the user control the `R`:

```python
def clear_coefficients(expr, rhs=S.Zero):
    """Return `p, r` where `p` is the expression obtained when Rational
    additive and multiplicative coefficients of `expr` have been stripped
    away in a naïve fashion (i.e. without simplification). The operations
    needed to remove the coefficients will be applied to `rhs` and returned
    as `r`.  If you want the solve 2 or more expressions like `expr = a`, 
    `expr = b`, etc..., it would be advantageous to provide a Dummy symbol
    for `rhs` and  simply replace it with `a`, `b`, etc... in `r`.

    Examples
    ========
    >>> from sympy.simplify.simplify import clear_coefficients  
    >>> from sympy.abc import x, y
    >>> from sympy import Dummy
    >>> clear_coefficients(2 + 4*y*(6*x + 3))  
    >>> (y*(2*x + 1), -1/6) 
    >>> r = Dummy('rhs')
    >>> clear_coefficients(4*y*(6*x + 3), r)
    (y*(2*x + 1), _rhs/12)
    >>> _[1].subs(r, -2)
    -1/6
    """
```

(and modify the function accordingly)